shuup/front/template_helpers/general.py

```diff
@@ -410,3 +410,10 @@ def can_toggle_all_seeing(context):
         # 'is all seeing' is purely person contact feature.
         return False
     return getattr(request.user, "is_superuser", False)
+
+
+@contextfunction
+def get_admin_url(context, object):
+    from shuup.admin.template_helpers.shuup_admin import model_url
+    return {"url": model_url(context, object),
+            "name": str(object)}
```

Comment:
this syntax looks weird.
you can either:

### Put everything in a single line
```py
return {"url": model_url(context, object), "name": str(object)}
# or 
return dict(url=model_url(context, object), name=str(object))
```

### Use multiple lines
```py
return {
    "url": model_url(context, object),
    "name": str(object)
}
# or 
return dict(
    url=model_url(context, object),
    name=str(object)
)
```
We use Django's coding styling (with few exceptions) and you can check it if in doubt:  https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/
Also follow PEP8:
https://www.python.org/dev/peps/pep-0008/